<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TemplatePage</class>
 <widget class="QWidget" name="TemplatePage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>300</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QTreeView" name="treeView">
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="loadButton">
       <property name="text">
        <string>Load Template From File</string>
       </property>
       <property name="icon">
        <iconset theme="application-x-archive">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="getNewButton">
       <property name="text">
        <string>Get More Templates</string>
       </property>
       <property name="icon">
        <iconset theme="get-hot-new-stuff">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="shareButton">
       <property name="text">
        <string>Share Templates</string>
       </property>
       <property name="icon">
        <iconset theme="get-hot-new-stuff">
         <normaloff/>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="extractButton">
       <property name="text">
        <string>Clone Template</string>
       </property>
       <property name="icon">
        <iconset theme="archive-extract"/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
